multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -23,6 +23,8 @@ import type { CurrentWalletRecord } from '@agoric/smart-wallet/src/smartWallet.j
 import type { QueryBalanceResponseSDKType } from '@agoric/cosmic-proto/cosmos/bank/v1beta1/query.js';
 import type { USDCProposalShapes } from '@agoric/fast-usdc/src/pool-share-math.js';
 
+const RELAYER_TYPE = process.env.RELAYER_TYPE;
```

Comment:
to make clear the name is unchanged,
```suggestion
const { RELAYER_TYPE }  = process.env;
```